cookbooks/fastscape_eroding_box.prm

```diff
@@ -0,0 +1,172 @@
+      set Surface resolution = 4
+
+      # Because we only have 1 level of refinement at the surface we set this to 0.
+      # If we had multiple resolutions, e.g. 3 different cell sizes at the surface,
+      # this would be set to 2 (number of cell sizes at surface - 1).
+      set Resolution difference = 0
+
+      # We set fastscape as 1 levels more resolved than ASPECT's initial global surface resolution.
+      set Additional fastscape refinement = 1
+
+      # Number of FastScape timesteps we want per ASPECT timestep.
+      set Number of steps = 5
+
+      # Set the maximum allowable FastScape timestep. If the (ASPECT timestep)/(Number of steps)
+      # is greater than this value, then the number of steps is automatically doubled.
+      # This is continued until the FastScape timestep is less than this value.
+      set Maximum timestep = 10000
+
+      # Seed number for initial topography noise
+      set Fastscape seed = 1000
+
+      # Because no boundaries are periodic, no flux, and no advection we don't need ghost nodes.
+      set Use ghost nodes = false
+      set Vertical exaggeration = 1
+
+      # Fix all boundaries in FastScape.
+      subsection Boundary conditions
+        set Bottom = 1
+        set Right  = 1
+        set Top    = 1
+        set Left   = 1
+      end
+   
+      # We use both SPL and diffusion. The bedrock deposition
+      # variable allows deposition of sediment, however, because
+      # the sediment variables are not set, they are equal to
+      # bedrock values.
+      subsection Erosional parameters
+        set Drainage area exponent = 0.4
+        set Bedrock diffusivity = 1e-2
+        set Bedrock river incision rate = 1e-4
+        set Slope exponent = 1
+        set Bedrock deposition coefficient = 1
+
+        # A negative value indicates varied flow. 10 is steepest descent, and 1 is uniform distribution.
+        set Multi-direction slope exponent = -1
+       end
+   end
+
+   set Additional tangential mesh velocity boundary indicators = left, right, front, back
+
+end
+
+subsection Gravity model
+  set Model name = vertical
+
+  subsection Vertical
+    set Magnitude = 0   # = Ra 
+  end
+end
+
+# Dimensionless
+subsection Material model
+  set Model name = simple
+
+  subsection Simple model
+    set Reference density             = 1
+    set Reference specific heat       = 1
+    set Reference temperature         = 0
+    set Thermal conductivity          = 1
+    set Thermal expansion coefficient = 1
+    set Viscosity                     = 1
+  end
+end
+
+
+# We also have to specify that we want to use the Boussinesq
+# approximation (assuming the density in the temperature
+# equation to be constant, and incompressibility).
+subsection Formulation
+  set Formulation = Boussinesq approximation
+end
+
+
+# We set our global refinement at the resolution level we want at the surface. As of now,
```

Comment:
What do you mean by `As of now`?

Reply:
I think I thought this could somehow change at some point. I've updated the comment.